spectral/manual-spectral.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/bash
```

Comment:
Just wondering if in the medium term this would become a task in the `package.json`?

Reply:
My plan was to use this script from the workflows to perform validations.
I think the eventual goal is to use the CLI to perform validations in CI - i.e. just run the calm cli against any patterns we have. That way we can get JSON schema checks too.

That of course means we need to be able to strip $refs out of the documents on the CLI too - which I'm taking a look at now.